File: 1-Two_Sum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int *twoSum(int* nums, int numsSize, int target) {
    int *arr = malloc(2 * sizeof(int));
    int i, j;
    int flag = 0;

    for(i = 0; i < numsSize; i++) {
        for(j = i+1; j < numsSize; j++) {
            if(nums[i] + nums[j] == target) {
                arr[0] = i;
                arr[1] = j;
                flag = 1;
                break;
            }
        }

        if (flag == 1)   break;
    }

    return arr;
}

File: 1-Two_Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """

        for id, num in enumerate(nums):
        	sub = target - num
        	if sub != num and sub in nums:
        		return [id, nums.index(sub)]